Tasks/ManualValidationV0/task.json

```diff
@@ -15,10 +15,9 @@
     ],
     "version": {
         "Major": 0,
-        "Minor": 241,
-        "Patch": 0
+        "Minor": 242,
+        "Patch": 1
```

Comment:
Do you have any reason to skip 0.242.0?

Reply:
I had bumped the task version again along with another commit I did. Updated the task version to 0.242.0